playbooks/services.yml

```diff
@@ -15,6 +15,8 @@
       tags: ['kube-prometheus']
     - role: kube_prometheus
       tags: ['kube-prometheus']
+    - role: kube_nginx_ingress_monitoring
```

Comment:
Would it make sense to just merge this into the `kube_nginx_ingress` role (maybe behind a flag), and move that one after `kube_prometheus`?

Reply:
But there is a dependency on `wait_prometheus_etcd` that will fail if kube_prometheus is not installed. And we can install the monitoring without nginx-ingress (just, that endpoints will be empty)